<paste log output from Kubernetes-mediated prediction, here>
Error from server (AlreadyExists): pods "mlapp" already exists
NAME                 TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
service/kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP   10m

NAME        READY   STATUS    RESTARTS   AGE
pod/mlapp   1/1     Running   0          4m16s
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000
Error from server (AlreadyExists): pods "mlapp" already exists
NAME                 TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
service/kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP   10m

NAME        READY   STATUS    RESTARTS   AGE
pod/mlapp   1/1     Running   0          4m16s
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000
voclabs:~/environment $ kubectl logs mlapp
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 179-328-287
[2023-03-04 18:59:48,948] INFO in app: JSON payload: 
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2023-03-04 18:59:48,962] INFO in app: Inference payload DataFrame: 
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2023-03-04 18:59:48,972] INFO in app: Scaling Payload: 
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2023-03-04 18:59:48,976] INFO in app: Output Prediction Value: 
[20.35373177134412]
127.0.0.1 - - [04/Mar/2023 18:59:48] "POST /predict HTTP/1.1" 200 -
